dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/scalaUtils/PythonOption.scala

```diff
@@ -0,0 +1,22 @@
+package me.shadaj.scalapy.tensorflow.api.scalaUtils
+
+import me.shadaj.scalapy.py
+import scala.language.implicitConversions
+
+class PythonOption[A] {
```

Comment:
Hmm `object PythonOption` and parametrize `option2PyOption`/`pyOption2Option` instead (possibly making them `def`s in the process)? This would look more natural/familiar

Reply:
If only it worked. :D 
I will try to get it done.